ktor-shared/ktor-websocket-serialization/common/src/io.ktor.websocket.serialization/WebsocketChannelSerialization.kt

```diff
@@ -0,0 +1,74 @@
+package io.ktor.websocket.serialization
```

Comment:
path of a file is now`src/io.ktor.websocket.serialization/WebsocketChannelSerialization.kt` - one folder, where package name is with dots, instead of folders tree :)